compiler/rustc_trait_selection/src/error_reporting/traits/fulfillment_errors.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-tidy-filelength
```

Comment:
we should move some stuff out of this file instead 🤔 probably as a separate PR

Reply:
Yeah I can do this as a seperate PR later